docs/Running-Tachyon-on-EC2-Yarn.md

```diff
@@ -0,0 +1,201 @@
+with Yarn on EC2 machines using the
+[Vagrant scripts](https://github.com/amplab/tachyon/tree/master/deploy/vagrant) that come with
+Tachyon.
+
+# Prerequisites
+
+**Install Vagrant and the AWS plugins**
+
+Download [Vagrant](https://www.vagrantup.com/downloads.html)
+
+Install AWS Vagrant plugin:
+
+```bash
+$ vagrant plugin install vagrant-aws
+$ vagrant box add dummy https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box
+```
+
+**Install Tachyon**
+
+Download Tachyon to your local machine, and unzip it:
+
+```bash
+$ wget http://tachyon-project.org/downloads/files/{{site.TACHYON_RELEASED_VERSION}}/tachyon-{{site.TACHYON_RELEASED_VERSION}}-bin.tar.gz
+$ tar xvfz tachyon-{{site.TACHYON_RELEASED_VERSION}}-bin.tar.gz
+```
+
+**Install python library dependencies**
+
+Install [python>=2.7](https://www.python.org/), not python3.
+
+Under `deploy/vagrant` directory in your home directory, run:
+
+```bash
+$ sudo bash bin/install.sh
+```
+
+Alternatively, you can manually install [pip](https://pip.pypa.io/en/latest/installing/), and then
+in `deploy/vagrant` run:
+
+```bash
+$ sudo pip install -r pip-req.txt
+```
+
+# Launch a Cluster
+
+To run a Tachyon cluster on EC2, first sign up for an Amazon EC2 account
+on the [Amazon Web Services site](http://aws.amazon.com/).
+
+Then create [access keys](https://aws.amazon.com/developers/access-keys/) and set shell environment
+variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` by:
+
+```bash
+$ export AWS_ACCESS_KEY_ID=<your access key>
+$ export AWS_SECRET_ACCESS_KEY=<your secret access key>
+```
+
+Next generate your EC2
+[Key Pairs](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html). Make sure to set
+the permissions of your private key file that only you can read it:
+
+```bash
+$ chmod 400 <your key pair>.pem
+```
+
+In the configuration file `deploy/vagrant/conf/ec2.yml`, set the value of `Keypair` to your keypair
+name and `Key_Path` to the path to the pem key.
+
+By default, the Vagrant script creates a
+[Security Group](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-network-security.html)
+named *tachyon-vagrant-test* at
+[Region(**us-east-1**) and Availability Zone(**us-east-1a**)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html).
+The security group will be set up automatically in the region with all inbound/outbound network
+traffic opened. You can change the security group, region and availability zone in `ec2.yml`.
+
+Now you can launch the Tachyon cluster with Hadoop2.4.1 as under filesystem in us-east-1a by running
+the script under `deploy/vagrant`:
+
+```bash
+$ ./create <number of machines> aws
+```
+
+# Access the cluster
+
+**Access through Web UI**
+
+After command `./create <number of machines> aws` succeeds, you can see two green lines like below
+shown at the end of the shell output:
+
+    >>> TachyonMaster public IP is xxx <<<
```

Comment:
These two lines have been changed, please refer to: https://github.com/amplab/tachyon/pull/1770/files#diff-1c8a2140e6601cebfefd4c9505a8b0d3R106